src/generator/css/base.less

```diff
@@ -776,5 +796,6 @@ body.post-page {
 
     img {
         @apply my-8;
+        display: none;
```

Comment:
Global display none for images under post-list? is this intentional? as row 661 is dealing with image visibility

Reply:
This is so the image doesnt show in the single post page. else the previous image visibility on line 661 doesnt handle it